Calibration.py

```diff
@@ -1,18 +1,22 @@
 import core
 import time
 import cwiid
-from lib_oled96 import ssd1306
 import os.path
 from ConfigParser import SafeConfigParser
 from enum import Enum
+# from lib_oled96 import ssd1306
+from core import ServoEnum
+try:
+    from collections import OrderedDict
+except ImportError:
+    from ordereddict import OrderedDict
 
-
-class CalibrationMode(Enum):
-    CM_NONE = 0
-    CM_LEFT = 1
-    CM_RIGHT = 2
-    CM_LEFT_AUX_1 = 3
-    CM_RIGHT_AUX_1 = 4
+# class CalibrationMode(Enum):
```

Comment:
Should be removed. Missed that one.

There were several areas removed due to improved handling of servo's in dictionary meant lots of stuff was no longer required to test which motor/servo was being dealt with.